client/python/gradio_client/client.py

```diff
@@ -110,6 +120,90 @@ def __init__(
         # Disable telemetry by setting the env variable HF_HUB_DISABLE_TELEMETRY=1
         threading.Thread(target=self._telemetry_thread).start()
 
+    @classmethod
+    def duplicate_from(
+        cls,
+        from_id: str,
+        to_id: str | None = None,
+        hf_token: str | None = None,
+        private: bool = True,
+        hardware: str | None = None,
+        secrets: Dict[str, str] | None = None,
+        max_workers: int = 40,
+        verbose: bool = True,
+    ):
+        """
+        Duplicates a Hugging Face Space under your account and returns a Client object
+        for the new Space. No duplication is created if the Space already exists in your
+        account (to override this, provide a new name for the new Space using `to_id`).
+        To use this method, you must provide an `hf_token` or be logged in via the Hugging
+        Face Hub CLI.
+
+        The new Space will be private by default and use the same hardware as the original
+        Space. This can be changed by using the `private` and `hardware` parameters. For
+        hardware upgrades (beyond the basic CPU tier), you may be required to provide
+        billing information on Hugging Face: https://huggingface.co/settings/billing
+
+        Parameters:
+            from_id: The name of the Hugging Face Space to duplicate, e.g. "abidlabs/whisper".
+            to_id: The name of the new Hugging Face Space to create, e.g. "whisper-duplicate". Do not add your Hugging Face username. If not provided, the new Space will be named "{you_hf_username}/{from_id}".
```

Comment:
I got confused by the docstring here. Are you able to duplicate into to an org?

Reply:
Yes, if you provide the HF_TOKEN corresponding to the org. What could make this clearer?

Reply:
It says

> The name of the new Hugging Face Space to create, e.g. "whisper-duplicate". Do not add your Hugging Face username.

Which made me think `gradio/whisper-duplicate` was not a valid value as it seems it only expects the name after the `/`

Reply:
Yes, that's correct. You should provide `whisper-duplicate`, not `gradio/whisper-duplicate`

Reply:
The username is automatically inferred from the HF_TOKEN

Reply:
I can change it so that you can provide one but it is ignored

Reply:
I think right now you are not able to clone to an org you are a member of?
Because the token will correspond to my personal namespace and not the one for the org I want.

Using the token for the org won't work because you need a personal token to create new repos

![image](https://user-images.githubusercontent.com/41651716/231557079-2ad4526d-c9b1-4a9e-8d50-659dbb84534c.png)

Reply:
Oh interesting, was not aware of that. In that case, let me change this

Reply:
Pushed a fix